src/containers/BrowseMoviesContainer.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+import BrowseMoviesPage from "../components/BrowseMoviesPage";
+import { getMoviesFromType, getGenreMovies } from "../api/APIUtils";
+
+class BrowseMoviesContainer extends Component {
+  state = { movies: [] };
+
+  componentDidMount() {
+    this.getMoviesFromTab();
+  }
+
+  componentDidUpdate() {
+    this.getMoviesFromTab();
```

Comment:
Don't call functions that change state in componentDidUpdate without any check, since it will be called infinity times